docs/release-notes/index.md

```diff
@@ -0,0 +1,114 @@
+---
+navigation_title: EDOT Python
+description: Release notes for Elastic Distribution of OpenTelemetry Python.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-sdk
+---
+
+# Elastic Distribution of OpenTelemetry Python release notes [edot-python-release-notes]
+
+Review the changes, fixes, and more in each version of Elastic Distribution of OpenTelemetry Python.
+
+To check for security updates, go to [Security announcements for the Elastic stack](https://discuss.elastic.co/c/announcements/security-announcements/31).
+
+% Release notes include only features, enhancements, and fixes. Add breaking changes, deprecations, and known issues to the applicable release notes sections.
```

Comment:
So deprecations and breaking changes should be added both here and in the other docs? What's the rationale of having them split in different files?

Reply:
Not here, only in the other docs